packer/builder/azure/utils/BuildAzureVmName.go

```diff
@@ -8,22 +8,52 @@ import (
 	"math/rand"
 	"os"
 	"time"
-	"sync"
 )
 
 const availableSymbols = "0123456789abcdefghijklmnopqrstuvwxyz"
-const allowedVmNameLength = 15
 
-var randmu sync.Mutex
+var pwSymbols = []string{
+	"abcdefghijklmnopqrstuvwxyz",
+	"ABCDEFGHIJKLMNOPQRSTUVWXYZ",
+	"0123456789",
+}
+
+const allowedVmNameLength = 15
 
 func BuildAzureVmNameRandomSuffix(prefix string) (suffix string) {
-	randmu.Lock()
```

Comment:
pid based seeding is very important to create truly random strings in case of concurrent calls to rand.Intn (#12)